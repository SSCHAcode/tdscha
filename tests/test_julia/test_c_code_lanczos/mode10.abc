2.1958244463998568e-07	4.0276024921473788e-11	1.5733889154384624e-02
3.6468697303092622e-06	8.3559039317101021e-07	9.0498801175471731e-07
3.0534049459196896e-06	5.0895292523202957e-07	2.4681927813924320e-06
5.9420421098876569e-07	1.3307855172936552e-06	2.5065197880233572e-07
3.2293720829509200e-06	7.2428970213566219e-07	6.5974434410687580e-07
