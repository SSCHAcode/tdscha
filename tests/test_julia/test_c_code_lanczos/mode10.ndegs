6
3
3
6
3
8
8
4
4
