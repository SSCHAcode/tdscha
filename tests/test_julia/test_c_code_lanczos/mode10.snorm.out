1.0000000000000000e+00
5.3914777797987923e-02
6.4329928008835919e-06
1.7142913899868234e-05
8.6067986924177952e-07
1.6493409833638794e-06
